routes/caniuse/index.js

```diff
@@ -41,5 +42,5 @@ async function route(req, res) {
 
   // cache for 24hours (86400 seconds)
   res.set("Cache-Control", "max-age=86400");
-  res.json(body);
+  res.send(body);
```

Comment:
`res.send()` ensures JS objects are sent has `application/json` and strings go as `text/html`

Reply:
oh nice... good that it knows how to detect the difference!